### ISSUE TITLE
False positive E1120 (no-value-for-parameter) using SQLAlchemy Table.insert

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#194]
Originally reported by: **Robbie Coomber (BitBucket: [robbie_c](http://bitbucket.org/robbie_c))**

---

I've attached sample code which exhibits this problem.

```
#!bash

$ pylint --version
pylint 1.1.0, 
astroid 1.0.1, common 0.61.0
Python 2.7.3 (default, Aug  9 2012, 17:23:57) 
[GCC 4.7.1 20120720 (Red Hat 4.7.1-5)]

$ python -c "import sqlalchemy; print sqlalchemy.__version__"
0.9.4

$ pylint --reports=no sqlalchemy_testcase.py
************* Module sqlalchemy_testcase
E: 20, 4: No value passed for parameter 'dml' in function call (no-value-for-parameter)
```

The insert function is decorated, and the decorator fills in the 'dml' parameter. The callee providing it is an error.

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/194
